packages/gluestack-ui/src/r5/renderer.ts

```diff
@@ -0,0 +1,8 @@
+import { FhirUIRenderer } from "@bonfhir/react/r5";
+
+import { GlueStackFhirValue } from "./data-display/index";
+
+export const GluestackUIRenderer: Partial<FhirUIRenderer> = {
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  FhirValue: GlueStackFhirValue as any,
```

Comment:
I will address the type stuff in later PRs because It was requested that I submit this PR as is.

Reply:
I don't understand why this is `any` here. Just remove it and you should be fine.